source/includes/wp-api-v3/_refunds.md

```diff
@@ -0,0 +1,158 @@
+					}
+				],
+				"sku": "",
+				"price": -9
+			}
+		],
+		"_links": {
+			"self": [
+				{
+					"href": "https://example.com/wp-json/wc/v3/orders/723/refunds/724"
+				}
+			],
+			"collection": [
+				{
+					"href": "https://example.com/wp-json/wc/v3/orders/723/refunds"
+				}
+			],
+			"up": [
+				{
+					"href": "https://example.com/wp-json/wc/v3/orders/723"
+				}
+			]
+		}
+	}
+]
+```
+
+#### Available parameters ####
+
+| Parameter        | Type    | Description                                                                                                                  |
+|------------------|---------|------------------------------------------------------------------------------------------------------------------------------|
+| `context`        | string  | Scope under which the request is made; determines fields present in response. Options: `view` and `edit`. Default is `view`. |
+| `page`           | integer | Current page of the collection. Default is `1`.                                                                              |
+| `per_page`       | integer | Maximum number of items to be returned in result set. Default is `10`.                                                       |
+| `search`         | string  | Limit results to those matching a string.                                                                                    |
+| `after`          | string  | Limit response to resources published after a given ISO8601 compliant date.                                                  |
+| `before`         | string  | Limit response to resources published before a given ISO8601 compliant date.                                                 |
+| `exclude`        | array   | Ensure result set excludes specific IDs.                                                                                     |
+| `include`        | array   | Limit result set to specific ids.                                                                                            |
+| `offset`         | integer | Offset the result set by a specific number of items.                                                                         |
+| `order`          | string  | Order sort attribute ascending or descending. Options: `asc` and `desc`. Default is `desc`.                                  |
+| `orderby`        | string  | Sort collection by object attribute. Options: `date`, `modified`, `id`, `include`, `title` and `slug`. Default is `date`.    |
+| `parent`         | array   | Limit result set to those of particular parent IDs.                                                                          |
+| `parent_exclude` | array   | Limit result set to all items except those of a particular parent ID.                                                        |
+| `dp`             | integer | Number of decimal points to use in each resource. Default is `2`.                                                            |
```

Comment:
Side note: should we perhaps use the full name `decimal_points`  instead of the abbreviation here?

Reply:
> should we perhaps use the full name decimal_points instead of the abbreviation here?

`dp` is the name of the parameter used in the REST API, so if we changed it in the docs, we'd need to update the endpoint as well. Replacing `dp` with `decimal_points` would be backwards-incompatible, but I guess we could add `decimal_points` as an alias of `dp`.

I agree `decimal_points` would have been a better name from the start, though. For now I think we should keep the docs and the endpoint as they are.

Reply:
>I agree decimal_points would have been a better name from the start, though. For now I think we should keep the docs and the endpoint as they are.

👍